ogr/abstract.py

```diff
@@ -1230,6 +1230,14 @@ def get_file_content(self, path: str, ref: str = "master") -> str:
         """
         raise NotImplementedError
 
+    def get_files(self, ref: str = "master") -> List[str]:
+        """
+        Get files of the project.
+
+        :return: [str]
```

Comment:
From the method name and dostring, I would not be sure, what will I get.

- Mentioning the `path` would be useful.
- Do I get all of the files or only the top directory,...?